SwiftFlow/CoreTypes/Store.swift

```diff
@@ -8,13 +8,22 @@
 
 import Foundation
 
+/** 
+Defines the interface of Stores in Swift Flow. `MainStore` is the default implementation of this
+interaface. Applications have a single store that stores the entire application state. 
```

Comment:
Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)